docs/pages/reference/backends.mdx

```diff
@@ -172,6 +172,143 @@ teleport:
      etcd_max_client_msg_size_bytes: 15728640
 ```
 
+## PostgreSQL (Preview)
+
+<Admonition type="warning" title="Preview">
+  The PostgreSQL cluster state and audit log storage is currently in Preview.
+</Admonition>
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  Before continuing, please make sure to take a look at the [Cluster State section](../architecture/nodes.mdx#cluster-state)
+  in Teleport Architecture documentation.
+</Admonition>
+
+Teleport can use [PostgreSQL](https://www.postgresql.org/) as a storage back-end
+to achieve high availability. You must take steps to protect access to
+PostgreSQL in this configuration because that is where Teleport secrets like
+keys and user records will be stored. The PostgreSQL backend supports two types
+of Teleport data:
+
+- Cluster state
+- Audit log events
+
+The PostgreSQL backend requires PostgreSQL 13 or later, and, for the cluster
+state only, the [`wal2json`](https://github.com/eulerto/wal2json) logical
```

Comment:
Can we add information/example on how to install this plugin and check whether it's installed?

Reply:
86b1e7a6ed8d62b09e9ca9c231d7120e539d1027 adds installation references, unfortunately plugin loading is just `dlopen`, so there is no way to list available plugins without actually attempting to load them.